src/core/plugins/samples/fn.js

```diff
@@ -480,17 +480,17 @@ export const sampleFromSchemaGeneric = (schema, config={}, exampleOverride = und
       } else {
         const toGenerateCount = schema.minProperties !== null && schema.minProperties !== undefined && propertyAddedCounter < schema.minProperties
           ? schema.minProperties - propertyAddedCounter
-          : 4
-        for (let i = 1; i < toGenerateCount; i++) {
+          : 3
+        for (let i = 0; i < toGenerateCount; i++) {
```

Comment:
@hkosova,

Wouldn't the following loop definition solve the same thing without need to change other code?

```js
for (let i = 1; i <= toGenerateCount; i++) {
``

Reply:
@char0n yes, and this was in the [original commit](https://github.com/swagger-api/swagger-ui/pull/7432/commits/0f94540ec5aefbe1307037f4f3e64ebdfedd4203#diff-879e3fb9c89a199d185458ea296ccf500d8589642c726e5d0e287dcbca7a0271), but @tim-lai previously asked to use a 0-based loop:

> More conventionally, the for loop should be written as `for (let i = 0; i < toGenerateCount)`

I can change back if needed - let me know.

<br/>

Note that the `"additionalPropNN"` names should remain 1-based to match the names generated by Swagger Inflector.

Reply:
@hkosova yes please, revert to original commit. The less code we modify the better the PR is.